src/dist/js/searchResults.js

```diff
@@ -9,3 +9,22 @@ function rowClick(event) {
     const url = window.location.pathname.replace('/_search', `/${identifier}`);
     window.open(url, '_blank');
 }
+
+const tableHeaders = document.querySelectorAll('th[data-field]');
+tableHeaders.forEach((th) => {
```

Comment:
Probably for future consideration, we should look into some JS grid/table libraries that could do some of the sorting/searching out of box for us.

Reply:
Agreed. I am looking forward to updating to a SPA framework and rethink many things like this.